@bazel/webpack/bin/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+nodejs_binary(
+    name = "webpack-worker",
+    data = [
+        "@npm//@bazel/worker",
```

Comment:
don't hard-code "npm"

maybe if this file is only used from the npm package you can just use a relative label
`//@bazel/worker`
and 
`//webpack-cli`

Reply:
This file is not used in the npm package as npm_install lays out something similar automatically. this is only used internally by the repository, not by the end-users.

Reply:
I have added these directories to avoid pkg_npm substitutions and fences so that I don’t have to specify which repository to get these packages from.